src/app/pages/profile/components/contact-information/contact-information.component.html

```diff
@@ -0,0 +1,38 @@
+<nb-card>
+    <nb-card-header>Contact Information</nb-card-header>
+    <nb-card-body>
+      <div class="row">
+        <div class="col-sm-6">
+          <div class="form-group">
+            <label for="inputEmail">Email</label>
+            <input type="email" class="form-control" id="inputEmail" placeholder="Email">
```

Comment:
We should fetch the user email from the `user` entity passed to the parent component.

Reply:
The User type doesn't contain an email property.  See above ^

Reply:
That's true, the User type in `@core/data` should be refactored in order to hold at least the essential information like the email address. Anyway, on this matter I would wait for the maintainer's feedback.